tfhe/src/shortint/client_key/mod.rs

```diff
@@ -242,6 +244,81 @@ impl ClientKey {
         }
     }
 
+    /// Compute a trivial shortint ciphertext with the dimension of the big LWE secret key from a
+    /// given value.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use tfhe::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS;
+    /// use tfhe::shortint::{gen_keys, Ciphertext};
+    ///
+    /// // Generate the client key and the server key:
+    /// let (cks, sks) = gen_keys(PARAM_MESSAGE_2_CARRY_2_KS_PBS);
+    ///
+    /// let msg = 1;
+    ///
+    /// // Trivial encryption
+    /// let ct1: Ciphertext = sks.create_trivial(msg);
+    ///
+    /// let ct_res = cks.decrypt(&ct1);
+    /// assert_eq!(1, ct_res);
+    /// ```
+    pub fn create_trivial(&self, value: u64) -> Ciphertext {
+        let modular_value = value as usize % self.parameters.message_modulus().0;
+        self.unchecked_create_trivial(modular_value as u64)
+    }
+
+    pub fn unchecked_create_trivial(&self, value: u64) -> Ciphertext {
+        let lwe_size = match self
+            .parameters
+            .pbs_parameters()
+            .unwrap()
+            .encryption_key_choice()
+        {
+            EncryptionKeyChoice::Big => self.large_lwe_secret_key().lwe_dimension().to_lwe_size(),
+            EncryptionKeyChoice::Small => self.small_lwe_secret_key().lwe_dimension().to_lwe_size(),
+        };
```

Comment:
parameters should already have a `encryption_lwe_dimension` function

Reply:
Done